aten/src/ATen/native/transformers/attention.cpp

```diff
@@ -115,27 +115,24 @@ Tensor bmm_nt(const Tensor& a, const Tensor& b) {
   return c_.view({a.size(0), a.size(1), a.size(2), b.size(2)});
 }
 
+void masked_softmax_dropout_cpu(
```

Comment:
Where is this function used?

Reply:
Good point. I dont think this is used anywhere, should be just redundant stuff when resolving conflict, will remove it.